src/sentry/static/sentry/app/views/settings/organizationTeams/teamMembers.tsx

```diff
@@ -167,6 +173,9 @@ class TeamMembers extends React.Component {
           const orgMember = this.state.orgMemberList.find(
             member => member.id === selection.value
           );
+          if (orgMember === undefined) {
```

Comment:
```suggestion
          if (!orgMember) {
```

Reply:
hmm, this might be preference, but I tend to like being explicit like this